odoo/models.py

```diff
@@ -3002,12 +3002,12 @@ def _update_field_translations(self, field_name, translations, digest=None):
         """
         self.ensure_one()
 
-        installed_lang = set(code for code, _ in self.env['res.lang'].get_installed())
-        missing_languages = set(translations) - installed_lang
-        if missing_languages:
+        installed_langs = set(code for code, _ in self.env['res.lang'].get_installed())
+        missing_langs = set(translations) - installed_langs - {'en_US'}
+        if missing_langs:
             raise UserError(
-                _("The following language is not activated: %(missing_names)s",
-                missing_names=', '.join(missing_languages))
+                _("The following languages are not activated: %(missing_names)s",
```

Comment:
It is not helpful, the problem is not relevant to the field. It is only relevant to the activated languages. and translations.
The only solution is to filter languages for `translations` before calling this function. Which means you should debug how the `translations` is generated rather than inside of this method.

As I mentioned in the https://github.com/odoo/odoo/pull/113888#discussion_r1127651727
If you don't think the user can trigger this error, I really prefer to drop inactivated translations and keep it silent.
Because
1. this problem is not fatal
2. user can never see it, so no need to raise user-error
3. we know what is the correct and expected behavior, `translate missing language`== `to do nothing`  is intuitive

Reply:
Actually, I prefer to make it silent, but I respect the `UserError` added by @flvr-odoo which is also acceptable.
Maybe I can refactor the `UserError` to `warning` in my another task with also touch this part of codes, if you hold the same idea.